### ISSUE TITLE
Pausing a node does not stop touch handling

### ISSUE THREAD

**Issue by Birkemose** [cocos2d/cocos2d-objc#1051]
I guess, that when you pause a node, it should stop responding to touches ...

**Comment by slembcke** [comment]
Hmm. I read that and immediately thought, "Oh, I suppose it should."

It's been stuck in my head though... I feel like there might be a lot of exceptions to that rule. Brought it up with @andykorth since he was sitting next to me. I think the consensus between us was that a lot of games have interactive elements when they are paused. You could probably split the hierarchy up to handle that nicely, but perhaps not always? Hmm...

**Comment by Birkemose** [comment]
Yeah, I guess under some (most?) circumstances, it would be counter-intuitive to disable touches.
Would be easy to set up a flag in the responder manager, but there probably will be very very few people ever having this problem.

**Comment by Panajev** [comment]
But surely you could tag nodes to be able to determine which are paused and which bode (property inherited from the parent node down) with the ability to force pause or un-pause all nodes.

When we pause a game we might have an interactive menu, at least to un-pause the game itself, but we shouldn't really just be able to touch any sprite not intentionally marked as such.

Still it is possible to put a layer over everything and filter all touches that way too...

[web [iPhone msg]]
